sentences/de/_common.yaml

```diff
@@ -74,7 +74,7 @@ lists:
         out: 1
   cover_states:
     values:
-      - in: "(offen|auf|oben)"
+      - in: "(offen|auf|oben|geöffnet)"
```

Comment:
could you also please add test sentences in `tests/de/cover_HassGetState.yaml` to cover this addition?